docs/deprecated_scss-linting/.stylelintscssrc

```diff
@@ -0,0 +1,194 @@
+    "stylelint-scss"
+  ],
+  "rules": {
+    "at-rule-blacklist": ["debug"],                                                                                 comments    ???
+    "at-rule-name-space-after": "always",                                                                           comments    stylelint-config-standard   ->  always-single-line
+    "declaration-no-important": true,                                                                               comments    ???
+    "declaration-property-value-blacklist": { "border": ["none"] },                                                 comments
+    "function-comma-space-after": "always",                                                                         comments    stylelint-config-standard   -> always-single-line
+    "function-url-no-scheme-relative": true,                                                                        comments
+    "function-url-quotes": "always",                                                                                comments
+    "indentation": 4,                                                                                               comments    stylelint-config-standard   -> 2
+    "max-nesting-depth": 3,                                                                                         comments
+    "no-duplicate-selectors": true,                                                                                 comments
+    "no-invalid-double-slash-comments": false,                                                                      added       stylelint-config-recommended -> true
+    "property-no-vendor-prefix": true,                                                                              comments
+    "selector-class-pattern": "^[a-z\\-_0-9]+$",                                                                    comments
+    "selector-max-id": 0,                                                                                           // nicht in stylelintrc -> Kein id Selektor ???
+    "selector-no-qualifying-type": true,                                                                            comments
+    "selector-pseudo-element-colon-notation": "single",                                                             // nicht in stylelintrc stylelint-config-standard -> double !!! ie8 support fällt weg
+    "string-no-newline": false,                                                                                     added       stylelint-config-recommended -> true
+    "string-quotes": "single",                                                                                      comments
+
+    "scss/at-extend-no-missing-placeholder": true,
+    "scss/at-import-no-partial-leading-underscore": true,
+    "scss/selector-no-redundant-nesting-selector": true,
+    "scss/at-function-pattern": "^[a-z\\-_0-9]+$",
+    "scss/at-mixin-pattern": "^[a-z\\-_0-9]+$",
+    "scss/dollar-variable-pattern": "^[a-z\\-_0-9]+$"
+  }
+}
+
```

Comment:
Same here: not needed anymore, please remove

Reply:
File removed.